helpers/stable_swap_math/mod.rs

```diff
@@ -749,7 +751,7 @@ mod tests {
         let token_0_in: u128 = 62463425433;
         let (amount_out, fee_out) = swap_to(0, token_0_in, 1, &reserves, &fees, amp_coef)
             .unwrap_or_else(|_| panic!("Should return SwapResult"));
-        let (amount_in, fee_in) = swap_from(1, amount_out, 0, &reserves, &fees, amp_coef)
+        let (amount_in, fee_in) = swap_from(0, amount_out, 1, &reserves, &fees, amp_coef)
```

Comment:
Why sudden change?

Reply:
It didn't matter because the reserves are equal here.